### PR TITLE
fix: removed dependency on appcelerator.com

### PR DESCRIPTION
**Pull request by cb1kenobi**
JIRA: https://jira.axway.com/browse/TIMOB-21146

### CODE REVIEW

File: lib/jdk.js [hunk 1]
```diff
@@ -206,7 +206,7 @@ exports.detect = function detect(config, opts, finished) {
 							)
 							+ __('The __JAVA_HOME__ environment variable must point to the JDK and not the JRE (Java Runtime Environment).') + '\n'
 							+ __('You may want to reinstall the JDK by downloading it from %s.',
-								'__http://appcelerator.com/jdk__')
+								'__http://www.oracle.com/technetwork/java/javase/downloads/index.html__')
```
Comment (ewanharris) [inline lib/jdk.js:209]: Is there a reason not to use HTTPS on the URL?

File: lib/jdk.js [hunk 2]
```diff
@@ -275,7 +275,7 @@ exports.detect = function detect(config, opts, finished) {
 					message: __('JDK (Java Development Kit) not installed.') + '\n'
 						+ __('If you already have installed the JDK, verify your __JAVA_HOME__ environment variable is correctly set.') + '\n'
 						+ __('The JDK is required for Titanium and must be manually downloaded and installed from %s.',
-							'__http://appcelerator.com/jdk__')
+							'__http://www.oracle.com/technetwork/java/javase/downloads/index.html__')
```
Comment (ewanharris) [inline lib/jdk.js:278]: Same here

**Review by ewanharris — COMMENTED** [review]
Could we use HTTPS?

File: lib/jdk.js (hunk 1 above)
Comment (cb1kenobi) [reply lib/jdk.js]: Fixed!

File: lib/jdk.js (hunk 2 above)
Comment (cb1kenobi) [reply lib/jdk.js]: Fixed!

**Pull request merged**